.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "projects/tuist/fixtures/tuist_plugin"]
 	path = projects/tuist/fixtures/tuist_plugin
 	url = https://github.com/fortmarek/ExampleTuistPlugin/
+[submodule "submodules/TuistCloud"]
```

Comment:
If it makes things any easier, the submodule can point to `Sources/TuistCloud`, not sure if we need this extra 
`submodules` directory

Reply:
I used a different directory because the cloned structure doesn't match the convention that SPM expects. The mapping between the convention and the submodule directory structure is done via symlinks.